packages/shared/src/api/urbit.ts

```diff
@@ -79,7 +78,7 @@ export function configureClient({
   clientInstance.on('fact', (fact) => {
     logger.log(
       'received message',
-      runIfDev(() => escapeLog(JSON.stringify(fact)))
+      runIfDev(() => escapeLog(JSON.stringify(fact)))()
```

Comment:
Curious why the second set of parens is necessary here -- afaik `runIfDev` should execute the function immediately in dev environment?